infra/cifuzz/fuzz_target.py

```diff
@@ -102,6 +101,12 @@ def fuzz(self):
     else:
       command += ['-v', '%s:%s' % (self.out_dir, '/out')]
 
+    # If corpus can be downloaded use it for fuzzing.
+    self.latest_corpus_path = self.clusterfuzz_deployment.download_corpus(
+        self.target_name, self.out_dir)
+    if self.latest_corpus_path:
+      command += ['-e', 'CORPUS_DIR=' + self.latest_corpus_path]
```

Comment:
We need to map our host path to the container path? i.e. map to a path starting with "/out" ?

Reply:
I see! It would still be good to do this regardless, rather than rely this assumption of the environment this code is running in. The class allows any out_dir, so we should handle it accordingly. Otherwise it might bite us later if we change this assumption, or if this class gets used elsewhere.

Reply:
Done.